.circleci/config.yml

```diff
@@ -79,7 +79,7 @@ workflows:
               img: [ "16.04", "18.04"]
               compiler: ["clang", "gcc"]
               hdf5: ["1.10.4"]
-              moab: ["5.1.0"]
+              moab: ["9c96d17"]
```

Comment:
I don't know enough about CI/yaml to know if this can be defined only once in this file and reused???